modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -306,8 +306,8 @@ public ObjectRelationship updateObjectRelationship(
 	}
 
 	private ObjectField _addObjectField(
-			User user, String name, long objectDefinitionId1,
-			long objectDefinitionId2, String type)
+			Map<Locale, String> labelMap, User user, String name,
```

Comment:
@danvb112 can you sort and put labelMap after user? 

Reason: We use to put audit fields (like companyId, user, etc) first

Thx :D